platforms/common/uORB/uORBDeviceNode.hpp

```diff
@@ -60,7 +60,7 @@ typedef void *uorb_cb_handle_t;
 #if defined(CONFIG_BUILD_KERNEL)
```

Comment:
I'd rather add it here

#if defined(CONFIG_BUILD_KERNEL) || defined(__PX4_POSIX)

Reply:
thanks. this is cleaner and wont affect any other build config.